core/math/fixed/fixed.odin

```diff
@@ -1,55 +1,61 @@
 package math_fixed
 
+import "core:intrinsics"
 import "core:math"
 import "core:strconv"
-import "core:intrinsics"
 _, _, _ :: intrinsics, strconv, math
 
-Fixed :: struct($Backing: typeid, $Fraction_Width: uint)
-	where
-		intrinsics.type_is_integer(Backing),
-		0 <= Fraction_Width,
-		Fraction_Width <= 8*size_of(Backing) {
-	i: Backing,
+Fixed :: struct($Backing: typeid,
+$Fraction_Width: uint,) where intrinsics.type_is_integer(Backing),
+	0 <=
+	Fraction_Width,
+	Fraction_Width <=
+	8 * size_of(Backing)
+{
+	i:
+	Backing,
 }
 
-Fixed4_4  :: distinct Fixed(i8, 4)
-Fixed5_3  :: distinct Fixed(i8, 3)
-Fixed6_2  :: distinct Fixed(i8, 2)
-Fixed7_1  :: distinct Fixed(i8, 1)
+Fixed4_4 :: distinct Fixed(i8, 4)
+Fixed5_3 :: distinct Fixed(i8, 3)
+Fixed6_2 :: distinct Fixed(i8, 2)
+Fixed7_1 :: distinct Fixed(i8, 1)
 
-Fixed8_8  :: distinct Fixed(i16, 8)
+Fixed8_8 :: distinct Fixed(i16, 8)
 Fixed13_3 :: distinct Fixed(i16, 3)
 
 Fixed16_16 :: distinct Fixed(i32, 16)
-Fixed26_6  :: distinct Fixed(i32,  6)
+Fixed26_6 :: distinct Fixed(i32, 6)
 
 Fixed32_32 :: distinct Fixed(i64, 32)
 Fixed52_12 :: distinct Fixed(i64, 12)
 
 
 init_from_f64 :: proc(x: ^$T/Fixed($Backing, $Fraction_Width), val: f64) {
 	i, f := math.modf(math.abs(val))
-	x.i  = Backing(f * (1<<Fraction_Width))
-	x.i &= 1<<Fraction_Width - 1
+	x.i = Backing(f * (1 << Fraction_Width))
+	x.i &= 1 << Fraction_Width - 1
 	x.i |= Backing(i) << Fraction_Width
 	if val < 0 {
 		x.i *= -1
 	}
 }
 
-init_from_parts :: proc(x: ^$T/Fixed($Backing, $Fraction_Width), integer, fraction: Backing) {
+init_from_parts :: proc(
+	x: ^$T/Fixed($Backing, $Fraction_Width),
+	integer, fraction: Backing,
+) {
 	i, f := math.modf(val)
-	x.i  = fraction
-	x.i &= 1<<Fraction_Width - 1
+	x.i = fraction
+	x.i &= 1 << Fraction_Width - 1
 	x.i |= integer
 }
 
 to_f64 :: proc(x: $T/Fixed($Backing, $Fraction_Width)) -> f64 {
 	sign := -1.0 if x.i < 0 else 1.0
 	num := math.abs(x.i)
 	res := f64(num >> Fraction_Width)
-	res += f64(num & (1<<Fraction_Width-1)) / f64(1<<Fraction_Width)
+	res += f64(num & (1 << Fraction_Width - 1)) / f64(1 << Fraction_Width)
```

Comment:
In fact there's a whole lot of reformatting going on that's nothing to do with fixing a bug.